### PR TITLE
Cortegeinfo

### PR DESCRIPTION
**Pull request by J4ken**
Check if i did the admin permissions correclty

### CODE REVIEW

File: test/fixtures/cortege_lineups.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (patriksletmo) [inline test/fixtures/cortege_lineups.yml]: If you're not using the fixture in a test, consider removing the fixture completely.

Comment (patriksletmo) [inline test/fixtures/cortege_lineups.yml]: I disagree. The name should however have the 's' somewhere else (all_cortege_lineups).

Comment (ljukas) [inline test/fixtures/cortege_lineups.yml]: That works aswell.

File: app/controllers/api/v1/cortege_lineups_controller.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  def show
+    require_admin_permission AdminPermission::LIST_CORTEGE_APPLICATIONS
+    cortege = CortegeLineup.find(params[:id])
+    render :json => cortege
+  end
+
+  def update
+    require_admin_permission AdminPermission::LIST_CORTEGE_APPLICATIONS
+    cortege = CortegeLineup.find(params[:id])
+    if cortege.update(item_params)
```
Comment (patriksletmo) [inline app/controllers/api/v1/cortege_lineups_controller.rb]: I beg to differ https://apidock.com/rails/ActiveRecord/Persistence/update

Comment (ljukas) [inline app/controllers/api/v1/cortege_lineups_controller.rb]: https://apidock.com/rails/ActiveRecord/Base/update/class
WHICH ONE IS IT?!

Comment (ljukas) [inline app/controllers/api/v1/cortege_lineups_controller.rb]: Never mind, you are correct, the one i found is deprecated... Sorry

**Pull request merged**